### PR TITLE
Add kernel profiles and css class to pages containers

### PR DESCRIPTION
**Pull request by boubaker**
When uninstalling some addons, the pages that defines portlets / applications defined by deleted addons gets buggy with error message 'Portlet not found'. Knowing that each addon defines its own Kernel profile when installed, this improvement makes it easy to disable a container with its applications in rendering phase, when the Kernel profile isn't present.
Another attribute is added in Container definition in pages.xml file that allows to define a list of CSS Classes to use in container Parent.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -824,7 +824,7 @@
               </property>
               <property>
                 <name>exo.profiles</name>
-                <value>${surefire.exo.profiles}</value>
+                <value>${surefire.exo.profiles},test</value>
```
Comment (rdenarie) [inline pom.xml:827]: Are you to put the value test ?

Comment (boubaker) [reply pom.xml:827]: This is just an Exo Kernel profile that is added to be used after in Surefire Unit Tests

**Review by azayati — APPROVED** [review]
LGTM

**Review by rdenarie — APPROVED** [review]

**Review by ahamdi — APPROVED** [review]
LGTM

**Pull request merged**